Source/WTF/wtf/text/StringView.cpp

```diff
@@ -447,6 +447,35 @@ int codePointCompare(StringView lhs, StringView rhs)
     return codePointCompare(lhs.characters16(), lhs.length(), rhs.characters16(), rhs.length());
 }
 
+template<typename CharacterType> static String makeStringBySimplifyingNewLinesSlowCase(const String& string, unsigned firstCarriageReturn)
+{
+    unsigned length = string.length();
```

Comment:
May be worth ASSERTing some sanity for `firstCarriageReturn`. Chances are this function is so much of a special case that no one will ever try to misuse it, so not feeling strongly about the need for sanity checks.

Reply:
I feel comfortable leaving this as is.